libs/coin-polkadot/src/types/model.fixture.ts

```diff
@@ -0,0 +1,109 @@
+import BigNumber from "bignumber.js";
+import { faker } from "@faker-js/faker";
+import { listCryptoCurrencies } from "@ledgerhq/cryptoassets/currencies";
+import { PolkadotAccount, PolkadotOperationExtra, PolkadotResources, Transaction } from "../types";
+import { getAbandonSeedAddress } from "../../../ledgerjs/packages/cryptoassets/lib";
+import { Operation } from "@ledgerhq/types-live";
+
+export function createFixtureAccount(account?: Partial<PolkadotAccount>): PolkadotAccount {
+  const currency = listCryptoCurrencies(true).find(c => c.id === "polkadot")!;
+
+  const polkadotResources: PolkadotResources = account?.polkadotResources || {
+    controller: undefined,
+    stash: account?.polkadotResources?.stash || undefined,
+    nonce: faker.number.int(100_000),
+    lockedBalance: new BigNumber(faker.string.numeric()),
```

Comment:
Here there might be a real life logic between lockedBalance,unlockedBalance, etc... that faker does not take into account